Makefile

```diff
@@ -39,11 +39,14 @@ GIT_SHA     ?= $(shell git update-index -q --refresh && \
 # Tools
 DOCKER      ?= docker
 DOCKER_RUN  ?= $(DOCKER) run --rm $(1) \
-               -e DOCKER= \
-               -w /go/src/$(GOMOD) \
-               -v $(WORKDIR):/go/src/$(GOMOD) \
-               $(REGISTRY)/$(2):$(IMAGE_TAG) \
-                 $(3)
+				-e DOCKER= \
+				-e GOOS=$(GOOS) \
+				-e GOARCH=$(GOARCH) \
+				-w /go/src/$(GOMOD) \
+				--platform linux/amd64 \
```

Comment:
Shouldn't the platform be based on the host?

Reply:
As of now, we only have a linux/amd64 docker image. This allows us to cross-compile for darwin/amd64 and darwin/arm64. You can also pass GOOS and GOARCH to the parent Makefile from the host to cross-compile for other platforms, for example:

make GOOS=linux GOARCH=loong64 kraft

Reply:
Nipick: Can you please keep the space-indentation as it was before. Keeps the git blame neater and the file well-aligned. Make shouldn't be complaining about these types of indentation.

Reply:
Done